profiling/performance_test_script.py

```diff
@@ -0,0 +1,11 @@
+import pandas as pd
```

Comment:
Could we add a header on top of every script to describe what each script is doing?